### ISSUE TITLE
Suggestion: color for ranges 

### ISSUE THREAD

**Issue by antonio1475** [danimart1991/pvpc-hourly-pricing-card#71]
Ability to customize ranges or thresholds for prices, with color.

For example, the background:
![image](https://github.com/user-attachments/assets/df727c74-5a25-4d2f-9ef1-4a7509b460d0)

Alternative: the graph line is colored instead:
![image](https://github.com/user-attachments/assets/90291b79-90b0-4c62-805a-ebdd4a0081d9)

Note: images badly done with Photopea, for illustrative purposes only.

Thanks 😄

**Comment by danimart1991** [comment]
Good idea, but I see some limitations here. 

The prices are so variant to stablish a common threshold in distinct days. How to configure it? 🤔

**Comment by antonio1475** [comment]
In my personal case: I'm interested in fixed ranges because for example periods below 0.10€ are a good time to do certain power consuming things like dishwasher, laundry, air conditioning... And if it's above 0.20€ it's best to turn on the heater instead of the AC heat.

Of course you already see this on the graph, colors are just to make it visually easier/faster 😅 

Another way is to make it relative to the max & min of the day...? 🤔  but that wouldn't really solve my use-case (eg if the day min is 0.12€ it would be green, but that doesn't mean that it's a great day for laundry.

_Maybe_ supporting templating in the configuration solves this: leave it up to the user to figure out the right value or formula. All the needed information is in the sensor.pvpc attributes

**Comment by orbitaloop** [comment]
Good idea. For me it would be super useful to have a background color representing the current price so we can see it from far away. That way, in the dashboard I can see from far away if this is a good time without having to check in detail. For example, when it's very expensive, put it in red, and when it's the cheapest, put it in green (with all the intermediate colors).
